controllers/userController.js

```diff
@@ -0,0 +1,31 @@
+const { catchAsync } = require('../utils/error');
+const userService = require('../services/userService');
+
+const signUp = catchAsync(async (req, res) => {
+  const { first_name, last_name, email, password } = req.body;
```

Comment:
```suggestion
  const { first_name: firstName, last_name: lastName, email, password } = req.body;
```
자바스크립트에서는 기본적으로 네이밍시 snake_case(단어사이를 언더바로 이어주는 방법)를 쓰지 않고, camelCase(소문자로 시작하여 대문자를 섞는 방법)를 사용하는 것이 코딩 컨벤션입니다. 구조분해 할당을 하면서 동시에 위와같이 변수 네이밍을 재설정 할 수 있으니 확인 후 반영해주세요~